index.html

```diff
@@ -0,0 +1,210 @@
+                <a class="nav-link link-menu" href="#">News</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu link-campaign" href="#">LIVE</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+    
+  <main class="main">
+    <div class="intro-main">
+      <div class="intro">
+        <h2 class="greet">EXPERIENCE THE SINGAPORE SMASH IN STYLE</h2>
+        <h1 class="title">Singapore Smash 2023<br>World Table Tennis 2023</h1>
+        <p class="description">
+          The first Grand Smash of the year, where the world’s Top 50 players will battle it out across nine days of exciting main draw action and Singapore will once again crown a champion. <br>
+
+New for 2023, experience an unparalleled atmosphere and an unbeatable view on all the action – THE SUITES at SINGAPORE SMASH – your bespoke, high-end table tennis experience.<br>
+
+Enjoy first-class entertainment and adrenaline-filled performances with staff or clients, all from an immersive fully serviced corporate box located above the player walkout runway.
+        </p>
+        <p class="date">2023.10.13(WED) ~ 15(FRI)</p>
+        <p class="place">@Singapore, Singapore.</p>
+      </div>
+    </div>
+
+    <section class="main-program" id="program">
+      <div class="main-title">
+        <h3 class="main-title-h3">THE SUITES</h3>
+      </div>
+      <div class="main-bar"></div>
+      <div class="main-activities">
+        <table class="table table-borderless t-main">
+          <tbody>
+            <tr class="main-row">
+              <td class="t-img"><img class="main-img" src="Images/main_01.png" alt="Speaker Icon"></td>
+              <td class="t-title">Full - Service Lounge</td>
+              <td class="t-description">Branded, private, full-service lounge with food beverages</td>
+            </tr>
+          </tbody>
+        </table>
+
+        <table class="table table-borderless t-main">
+          <tbody class="main-column">
+            <tr class="main-row">
+              <td class="t-img"><img class="main-img" src="Images/main_02.png" alt="Exhibit Icon"></td>
+              <td class="t-title">Unique Experiences</td>
+              <td class="t-description">Visit legends and current stars in the arena.</td>
+            </tr>
+          </tbody>
+        </table>
+  
+        <table class="table table-borderless t-main">
+          <tbody>
+            <tr class="main-row">
+              <td class="t-img"><img class="main-img" src="Images/main_03.png" alt="Forum Icon"></td>
+              <td class="t-title">Special Perks</td>
+              <td class="t-description">Backstage tour, training room, memorabilia and more.</td>
+            </tr>
+          </tbody>
+        </table>
+
```

Comment:
- [ ] From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see, your `SUITES` section suppose to have 5 categories of containers, your images are not showing, you are missing the `SEE THE WHOLE PROGRAM` text and you have no background design which makes your design not matching [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). So please make sure that you adjust the style of your design to match **_exactly_** like Cindy's design :+1: 

### Your design
![image](https://user-images.githubusercontent.com/92303891/227747868-399a343f-1ec6-47cb-8377-994aadc96d43.png)

### Cindy's design
![Screenshot from 2023-03-25 23-58-15](https://user-images.githubusercontent.com/92303891/227748403-3bcf03b9-1880-414a-9976-d2f5b2025a4c.png)